esmvaltool/recipes/recipe_miles_regimes.yml

```
@@ -0,0 +1,78 @@
+###############################################################################
+## namelist_miles.xml
+##
+## Description
+## Namelist to compute Weather Regimes using the MiLES package by P. Davini (ISAC-CNR)
+## MiLES (Mid-Latitude Evaluation System) v0.31
+##
+## Authors
+## Paolo Davini (ISAC-CNR, Italy - p.davini@isac.cnr.it)
+## J. von Hardenberg (ISAC-CNR, Italy - j.vonhardenberg@isac.cnr.it)
+## E. Arnone (ISAC-CNR, Italy - e.arnone@isac.cnr.it)
+##
+## Project
+## Copernicus C3S-MAGIC (C3S 34a Lot2) 
+##
+## References:
+## https://github.com/oloapinivad/MiLES
+## S. Corti, F. Molteni and T. N. Palmer, 1999 
+##   Signature of recent climate change in frequencies of natural 
+##   atmospheric circulation regimes". Nature 398, 799-802
+##
+## This namelist is part of the ESMValTool
+################################################################################
+#
+# mask_landocean: false
+# multi_model_statistics: false
+---
+documentation:
+  description: |
+    Recipe for computing and plotting weather regimes using 
+    the MiLES (Mid-Latitude Evaluation System) package
+
+  authors:
+    - hard_jo
+    - davi_pa
+    - arno_en
+
+  references:
+    - davini18
+    - corti99nat
+
+  projects:
+    - c3s-magic
+
+datasets:
+   - {dataset: EC-EARTH,    project: CMIP5,  exp: historical,  ensemble: r1i1p1,  start_year: 1980,  end_year: 1989}
+   - {dataset: ERA-Interim, project: OBS, type: reanaly, version: 1, tier: 3, start_year: 1980,  end_year: 1989 }
+     #- {dataset: ERA-Interim, project: OBS, type: reanaly, version: 1, tier: 3, start_year: 1980,  end_year: 1989, path: "/work/users/jost/esmvaltool2/input/OBS" }
+
+preprocessors:
+    preproc1:
+      extract_levels:
+        levels: 50000
+        scheme: nearest
+      regrid:
+        target_grid: 2.5x2.5
+        scheme: area_weighted
+      extract_region:
+        start_longitude: 0.
+        end_longitude: 360.
+        start_latitude: 1.25
+        end_latitude: 90.
+
+diagnostics:
+
+  miles_diagnostics:
+    description: MiLES Weather Regimes Diagnostics 
+    variables:
+      zg:
+        preprocessor: preproc1
+        mip: day
+        field: T2Ds
```

I think this should be a 3 dimensional field, because you're using the extract_levels preprocessor function and it does not make any sense to do that on a field that does not have a vertical coordinate.
```suggestion
        field: T3D
```

---

Yes, fixed.